### PR TITLE
Update CPI docstring

### PR DESCRIPTION
**Pull request by jmaslek**
close my jira ticket with better desciption.

### CODE REVIEW

File: openbb_platform/core/openbb_core/provider/standard_models/cpi.py [hunk 1]
```diff
@@ -80,9 +80,11 @@ class ConsumerPriceIndexQueryParams(QueryParams):
...
         + """
     Options:
-    - `growth_previous`: growth from the previous period
-    - `growth_same`: growth from the same period in the previous year
-    - `index_2015`: index with base year 2015.""",
+    - `growth_previous`: percent growth from the previous period.
+      If monthly data, this is month-over-month, etc
+    - `growth_same`: percent growth from the same period in the previous year.
+      if looking at monthly data, this would be year-over-year, etc.
+    - `index_2015`: rescaled index value, such that the value in 2015 is 100.""",
```
Comment (deeleeramone) [inline openbb_platform/core/openbb_core/provider/standard_models/cpi.py:87]: I think we need to make a grammar style choice.

'all lower case.  even after a period.' 
'Traditional capitalization for all phrases.'

I don't have a horse in the race, but am just looking for a definitive choice.

Comment (jmaslek) [reply openbb_platform/core/openbb_core/provider/standard_models/cpi.py:87]: Could not agree more.  Lets stick with (at least here), the way everyone learns to write sentences

Comment (deeleeramone) [reply openbb_platform/core/openbb_core/provider/standard_models/cpi.py:87]: One more to go on line 87. 😀

**Review by deeleeramone — APPROVED** [review]

**Review by piiq — APPROVED** [review]
Perfect, thank you

**Pull request merged**